Sources/BaggageContext/Context.swift

```diff
@@ -0,0 +1,287 @@
+    ///
+    /// ### Usage in frameworks and libraries
+    /// This function is really only intended to be used frameworks and libraries, at the "top-level" where a request's,
+    /// message's or task's processing is initiated. For example, a framework handling requests, should create an empty
+    /// context when handling a request only to immediately populate it with useful trace information extracted from e.g.
+    /// request headers.
+    ///
+    /// ### Usage in applications
+    /// Application code should never have to create an empty context during the processing lifetime of any request,
+    /// and only should create contexts if some processing is performed in the background - thus the naming of this property.
+    ///
+    /// Usually, a framework such as an HTTP server or similar "request handler" would already provide users
+    /// with a context to be passed along through subsequent calls.
+    ///
+    /// If unsure where to obtain a context from, prefer using `.TODO("Not sure where I should get a context from here?")`,
+    /// such that other developers are informed that the lack of context was not done on purpose, but rather because either
+    /// not being sure where to obtain a context from, or other framework limitations -- e.g. the outer framework not being
+    /// context aware just yet.
+    public static func background(logger: Logger) -> Context {
+        return .init(baggage: .background, logger: logger)
+    }
+
+    /// A baggage context intended as a placeholder until a real value can be passed through a function call.
+    ///
+    /// It should ONLY be used while prototyping or when the passing of the proper context is not yet possible,
+    /// e.g. because an external library did not pass it correctly and has to be fixed before the proper context
+    /// can be obtained where the TO-DO is currently used.
+    ///
+    /// ### Crashing on TO-DO context creation
+    /// You may set the `BAGGAGE_CRASH_TODOS` variable while compiling a project in order to make calls to this function crash
+    /// with a fatal error, indicating where a to-do baggage context was used. This comes in handy when wanting to ensure that
+    /// a project never ends up using with code initially was written as "was lazy, did not pass context", yet the
+    /// project requires context passing to be done correctly throughout the application. Similar checks can be performed
+    /// at compile time easily using linters (not yet implemented), since it is always valid enough to detect a to-do context
+    /// being passed as illegal and warn or error when spotted.
+    ///
+    /// - Parameters:
+    ///   - reason: Informational reason for developers, why a placeholder context was used instead of a proper one,
+    /// - Returns: Empty "to-do" baggage context which should be eventually replaced with a carried through one, or `background`.
+    public static func background(label loggerLabel: String) -> Context {
```

Comment:
I am not sure this adds tons of value, seems like pure sugar?

Reply:
Yeah, it's just sugar over the `.background(Logger("..."))` -- sure that we don't want the sugar for this?

I.e.

```
let context: Context = .background(Logger("background"))
let context: Context = .background(label: "background")
```


happy to remove

Reply:
Removed